File: ScalaPractise/src/GreenHand/Test1.scala
package GreenHand

object Test1 {
  def main(args:Array[String]){
    val a = 123l;
    println(a);

    // scala中是不允许声明变量是不进行初始化的。但是java中是可以的
    // var l;      // 这种方式是错误的
    var la = None;
    println(la);

    // 初始化一个变量时，如果该变量未赋初始值，一定要指定起数据类型
    // var an:Int = null
    var myVar:Int=10
    var an:Int=1
    an = 200
    println(an)

    //
    var c:Int=2
    c = 10
    println(c)
  }

}

File: JavaPractise/ExtractTabledataFromPDF/test/Test1.java
import org.apache.pdfbox.pdmodel.PDDocument;
import org.apache.pdfbox.text.PDFTextStripper;

import java.io.File;
import java.io.IOException;

public class Test1 {
    public static void main(String args[]){
        File pdffile = new File("/home/huabingood/沈之春重疾10万.pdf");

        PDDocument document = null;

        try {
            document = PDDocument.load(pdffile);

            int totalPage = document.getNumberOfPages();

            PDFTextStripper stripper = new PDFTextStripper();

            stripper.setSortByPosition(true);
            stripper.setStartPage(6);
            stripper.setEndPage(6);


            String content = stripper.getText(document);

            System.out.println(content);

        } catch (IOException e) {
            e.printStackTrace();
        }
    } // main
}
